src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java

```diff
@@ -21,9 +22,24 @@ public PizzaStylesApiV1Controller(PizzaStyleService pizzaStyleService) {
   }
 
   @GetMapping
-  public Map<String, List<PizzaStyle>> getAllPizzaStyles() {
-    Map<String, List<PizzaStyle>> pizzaStylesMap = new HashMap<>();
+  public Map<String, Iterable<PizzaStyle>> getAllPizzaStyles() {
+    Map<String, Iterable<PizzaStyle>> pizzaStylesMap = new HashMap<>();
     pizzaStylesMap.put("pizzaStyles", pizzaStyleService.findAll());
     return pizzaStylesMap;
   }
+
+  @GetMapping("/{id}")
+  public Map<String, PizzaStyle> getById(@PathVariable Integer id){
+    Map<String, PizzaStyle> map = new HashMap<>();
+
+    if(pizzaStyleService.findById(id).isPresent()){
+      PizzaStyle pizzaStyle = pizzaStyleService.findById(id).get();
```

Comment:
Instead of having to call `pizzaStyleService.findById(id)` on 35 and again on 36. Let's first set it to a variable before this conditional and then check if that Optional object that is returned `isPresent()` and if so we can get the actual value with `.get()`